### PR TITLE
chore: migrating + seeding

### PR DESCRIPTION
**Pull request by ChienNQuang**
- setup migrating everytime app is running
- setup base seeding to add an admin user

### CODE REVIEW

File: src/Api/Common/Serilogger.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ public static class Serilogger
         var applicationName = context.HostingEnvironment.ApplicationName?.ToLower().Replace(".", "-");
         var environmentName = context.HostingEnvironment.EnvironmentName ?? "Development";
 
-        // Configure structured loggin
```
Comment (kaitoz11) [inline src/Api/Common/Serilogger.cs:14]: I'm not laughing, ok!

**Review by kaitoz11 — COMMENTED** [review]
Its okay to merge, but dont you think we should wait until @Vzart done?

**Review by StarryFolf — APPROVED** [review]
cool

File: src/Infrastructure/Persistence/ApplicationDbContextSeed.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            logger.Error(ex, "An error occurred while seeding the database.");
+            throw;
+        }
+    }
+
+    private async Task TrySeedAsync(ApplicationDbContext context, IConfiguration configuration)
+    {
+        // Note: uncomment this
+    //     // Default roles
+    //     var administratorRole = "Administrator";
```
[the remark this answers is not in the document]
Comment (ChienNQuang) [reply src/Infrastructure/Persistence/ApplicationDbContextSeed.cs:26]: that'll be a TODO, I was too lazy then

**Comment by ChienNQuang** [comment]
> Its okay to merge, but dont you think we should wait until @Vzart done?

why?

**Review by Vzart — APPROVED** [review]
Approved, nothing seem wrong

**Review by kaitoz11 — APPROVED** [review]

**Pull request merged**